src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -198,7 +198,7 @@ export default {
 		async getSecondCategoryData() {
 			this.secondCategoryLoans = await this.getMatchedLoans();
 			this.matchedLoansTotal = this.secondCategoryLoans.length;
-			if (this.matchedLoansTotal === 0) this.secondCategoryLoans = await this.getExpiringSoonAlmostFundedCombo(); // eslint-disable-line max-len
+			if (this.matchedLoansTotal < 3) this.secondCategoryLoans = await this.getExpiringSoonAlmostFundedCombo(); // eslint-disable-line max-len
```

Comment:
We actually want to combine the matched with the fallback. So the first 1-2 loans are matched, and the rest are the fallback. Here we are replacing the matched loans results with the fallback.

Reply:
oh, I see the note 🤦‍♂️, ty